runtime/onert/frontend/circle_traininfo/include/traininfo_loader.h

```diff
@@ -27,7 +27,7 @@ namespace train
 namespace traininfo_loader
 {
 
-static constexpr char *const TRAININFO_METADATA_NAME = "CIRCLE_TRAINING";
+static const constexpr char *const TRAININFO_METADATA_NAME = "CIRCLE_TRAINING";
```

Comment:
@zetwhite May I ask why `const` is added here?

Reply:
About this part, I made a separate PR. 
Could you check this one? https://github.com/Samsung/ONE/pull/12410